Source/Core/Core/Boot/Boot.cpp

```diff
@@ -80,6 +80,12 @@ void CBoot::Load_FST(bool _bIsWii)
   DVDRead(fst_offset << shift, arena_high, fst_size << shift, _bIsWii);
   Memory::Write_U32(arena_high, 0x00000038);
   Memory::Write_U32(max_fst_size << shift, 0x0000003c);
+
+  if (_bIsWii)
+  {
+    // the apploader changes IOS MEM1_ARENA_END too
+    Memory::Write_U32(arena_high, 0x00003110);
```

Comment:
I haven't really looked into what the apploader really does (by REing it), but this seems consistent with what I've seen the apploader do while working on the ES_Launch fix.